diff --git a/dist/Container/index.js b/dist/Container/index.js
index 7c51a160a5379fab20495cf12b29dfe6fb12e861..ea4d2add6e29ca98d04ab424adf692292aabdc54 100644
--- a/dist/Container/index.js
+++ b/dist/Container/index.js
@@ -1,4 +1,5 @@
 "use strict";
+console.log("Container created")
 var __extends = (this && this.__extends) || (function () {
     var extendStatics = function (d, b) {
         extendStatics = Object.setPrototypeOf ||
